app/src/main/res/layout/activity_login.xml

```diff
@@ -124,116 +126,118 @@
+        android:text="@string/option_login"
         android:textColor="@color/tertiary"
+        android:textSize="@dimen/main_activity_txt_size"
         android:textStyle="bold"
-        android:gravity="center"
-        android:textSize="16sp"
-        />
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/btn_login"
+        tools:ignore="TextContrastCheck" />
 
     <LinearLayout
         android:id="@+id/btns_apps"
-        android:layout_width="264dp"
-        android:layout_height="70dp"
+        android:layout_width="@dimen/main_linearL_width_264dp"
+        android:layout_height="@dimen/main_linearL_height_70dp"
         android:orientation="horizontal"
-        tools:layout_editor_absoluteX="17dp"
-        tools:layout_editor_absoluteY="444dp"
         app:layout_constraintTop_toBottomOf="@id/txt_opcions"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
-        android:paddingEnd="25dp"
-        android:paddingStart="25dp"
-        android:paddingTop="10dp"
+        android:paddingEnd="@dimen/main_padding_25dp"
+        android:paddingStart="@dimen/main_padding_25dp"
+        android:paddingTop="@dimen/main_activity_margin_top"
         android:gravity="center"
         >
 
         <Button
             android:id="@+id/btn_apple"
-            android:layout_width="50dp"
-            android:layout_height="50dp"
-            android:layout_margin="15dp"
+            android:layout_width="@dimen/main_btn_options_50dp"
+            android:layout_height="@dimen/main_btn_options_50dp"
+            android:layout_margin="@dimen/main_btn_margin_15dp"
             android:background="@drawable/btn_icon_ios"
-            tools:ignore="SpeakableTextPresentCheck" />
+            tools:ignore="SpeakableTextPresentCheck"
+            style="?android:attr/buttonBarButtonStyle" />
 
         <Button
             android:id="@+id/btn_facebook"
-            android:layout_width="50dp"
-            android:layout_height="50dp"
-            android:layout_margin="8dp"
+            android:layout_width="@dimen/main_btn_options_50dp"
+            android:layout_height="@dimen/main_btn_options_50dp"
+            android:layout_margin="@dimen/dp8"
             android:background="@drawable/btn_icon_fb"
-            tools:ignore="SpeakableTextPresentCheck,SpeakableTextPresentCheck" />
+            tools:ignore="SpeakableTextPresentCheck,SpeakableTextPresentCheck"
+            style="?android:attr/buttonBarButtonStyle" />
 
         <Button
             android:id="@+id/btn_google"
-            android:layout_width="50dp"
-            android:layout_height="50dp"
-            android:layout_margin="15dp"
+            android:layout_width="@dimen/main_btn_options_50dp"
+            android:layout_height="@dimen/main_btn_options_50dp"
+            android:layout_margin="@dimen/main_btn_margin_15dp"
```

Comment:
remember not to tie dimens to specific screens when they can be reusable

Reply:
i'll implement it, thanks